FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -850,6 +850,16 @@ NS_SWIFT_NAME(Auth)
  */
 - (BOOL)canHandleNotification:(NSDictionary *)userInfo API_UNAVAILABLE(macos, tvos, watchos);
 
+/** @fn revokeToken:Completion
+    @brief Revoke the given token (currently only supports Apple issued access token, refresh token
```

Comment:
The API review claims all tokens are supported.

Reply:
Thanks for catching this.
Technically, our backend accepts all 3 types of tokens, however on Apple platforms, the developer only gets authorization code (refresh token and access token will be returned in Web and Android flows).
I suggest we should change the comments here. Let me confirm with the api council.

Reply:
Confirmed with the api council that it's ok with changing the comments here.